src/vaadin-overlay.html

```diff
@@ -907,15 +982,30 @@
         }
       }
 
+      /**
+       * @param {Element} element
+       * @return {boolean}
+       * @protected
+       */
       _isFocused(element) {
         return element && element.getRootNode().activeElement === element;
       }
 
+      /**
+       * @param {Element[]} elements
+       * @return {number}
+       * @protected
+       */
       _focusedIndex(elements) {
         elements = elements || this._getFocusableElements();
         return elements.indexOf(elements.filter(this._isFocused).pop());
       }
 
+      /**
+       * @param {number} increment
+       * @param {number} index
```

Comment:
This also seems to take `undefined` (there's a special case for it) so you can call it also by providing only single param (`increment`).
```suggestion
       * @param {number | undefined} index
```

Reply:
Changed.